packages/client/src/bags/__tests__/putBagPromocodes.test.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  mockBagId,
+  mockBagPromocodesData,
+  mockBagPromocodesResponse,
```

Comment:
In my point of view, we can create a new fixtures file for promocodes and separate them from the bag.

Reply:
I thought about that too but promo codes besides bag items and bag operation are something small and that depends on the bag so I think it might be better to be inside the bag fixtures. Is it ok for you?

Reply:
ok, fine!